integrations/amazon_bedrock/pyproject.toml

```diff
@@ -42,7 +42,9 @@ root = "../.."
 git_describe_command = 'git describe --tags --match="integrations/amazon_bedrock-v[0-9]*"'
 
 [tool.hatch.envs.default]
+installer = "uv"
 dependencies = [
+  "setuptools",
```

Comment:
This is needed to generate docs.
If you think it's better, we can add this as a dependency of `haystack-pydoc-tools`.

Reply:
done in https://github.com/deepset-ai/haystack-pydoc-tools/pull/5
released new version: https://pypi.org/project/haystack-pydoc-tools/0.3.2/